### ISSUE TITLE
6 Speed Fan

### ISSUE THREAD

**Issue by zaackyb21** [aarons22/homebridge-bond-v2#5]
Would it be possible to update the logic for the fan speed to check for the max speed of a fan and run the case statement logic based on that? I have a fan that supports six speeds so this would be helpful. I am sure there is a better way to do it but I have a limited coding background.

        if (device.maxSpeed == 6) {
          switch (speed) {
              case 16:
                  return commands[0];
              case 32:
                  return commands[1];
              case 48:
                  return commands[2];
              case 64:
                  return commands[3];
              case 80:
                  return commands[4];
              case 96:
                  return commands[5];    
              default:
                  return bond.powerOffCommand(device);
          }
      } else {
        switch (speed) {
            case 33:
                return commands[0];
            case 66:
                return commands[1];
            case 99:
                return commands[2];   
            default:
                return bond.powerOffCommand(device);
        }
      }

**Comment by aarons22** [comment]
Yeah I have a note in my code somewhere to support more than just 3 speeds. I've added it to the feature backlog!
